frontend/src/containers/signup-page/signup.page.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { useForm } from "react-hook-form";
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import styles from './styles.module.scss';
+import { RegisterPayloadKey, validationUserSchema } from 'healthcare-shared'
```

Comment:
It would be better to use proxy import
Firstly you need to import it in the frontend application
// frontend/validation-schema/user/index.ts
export { validationUserSchema } from 'healthcare-shared/validation-schema'

// frontend/src/containers/signup-page/signup.page.tsx
import { validationUserSchema } from 'validation-schema'

Do the same for enums and other things from the shared folder